app/models/service_template.rb

```diff
@@ -380,6 +380,8 @@ def self.create_from_options(options)
   private_class_method :create_from_options
 
   def provision_request(user, options = nil, request_options = nil)
+    request_options ||= {}
+    request_options[:submit_workflow] = true
```

Comment:
Let's change the method definition to use `request_options = {}`

Then you do not need the first line to initialize it.